.github/workflows/uffizzi-preview-deploy.yml

```diff
@@ -82,21 +82,21 @@ jobs:
     name: Use Remote Workflow to Preview on Uffizzi
     needs:
       - cache-compose-file
-    uses: UffizziCloud/preview-action/.github/workflows/reusable.yaml@desc
+    uses: UffizziCloud/preview-action/.github/workflows/reusable.yaml@v2
     with:
       # If this workflow was triggered by a PR close event, cache-key will be an empty string
       # and this reusable workflow will delete the preview deployment.
       compose-file-cache-key: ${{ needs.cache-compose-file.outputs.compose-file-cache-key }}
       compose-file-cache-path: docker-compose.rendered.yml
       server: https://app.uffizzi.com
       pr-number: ${{ needs.cache-compose-file.outputs.pr-number }}
-      description: |
-        The meilisearch preview environment contains a web terminal from where you can run the
-        `meilisearch` command. You should be able to access this instance of meilisearch running in
-        the preview from the link Meilisearch Endpoint link given below.
+      # description: |
+      #   The meilisearch preview environment contains a web terminal from where you can run the
+      #   `meilisearch` command. You should be able to access this instance of meilisearch running in
+      #   the preview from the link Meilisearch Endpoint link given below.
          
-        Web Terminal Endpoint : ${{ needs.cache-compose-file.outputs.expected-url }}
-        Meilisearch Endpoint : ${{ needs.cache-compose-file.outputs.expected-url }}/meilisearch
+      #   Web Terminal Endpoint : ${{ needs.cache-compose-file.outputs.expected-url }}
+      #   Meilisearch Endpoint : ${{ needs.cache-compose-file.outputs.expected-url }}/meilisearch
```

Comment:
Why do you comment this?
Also looks like it does not work currently

<img width="971" alt="Capture d’écran 2023-02-15 à 15 40 00" src="https://user-images.githubusercontent.com/20380692/219058828-8dc9eb58-08c7-46e2-9b01-810dd429afa8.png">